core/templates/pages/teach-page/teach-page.component.html

```diff
@@ -31,66 +31,66 @@ <h2>Teach with Oppia</h2>
               The story-based lessons we're creating relate mathematics to a student's real-life experience (similar to this <a href="https://www.oppia.org/explore/DIWZiVgs0km-?collection_id=4UgTQUc1tala">sample lesson</a> on Fractions). We want them to be demonstrably effective, to represent the best that can be done with online teaching, and to be available to anyone free-of-charge. We also plan to improve the lessons over time, so that they're as useful for as many students around the world as possible.
             </p>
             <p>
-              The <a ng-href="<[$ctrl.TEACH_FORM_URL]>"><em>Teach with Oppia</em></a> program invites applications from volunteers who can demonstrate a passion and aptitude for helping students learn basic mathematics. Participants will receive personalized feedback on the lessons they create, as well as assistance with playtesting and promoting their lessons. They'll also be able to connect with other like-minded creators, as well as the Oppia development team, to ensure their creations are well-realized.
+              The <a href="{{ TEACH_FORM_URL }}"><em>Teach with Oppia</em></a> program invites applications from volunteers who can demonstrate a passion and aptitude for helping students learn basic mathematics. Participants will receive personalized feedback on the lessons they create, as well as assistance with playtesting and promoting their lessons. They'll also be able to connect with other like-minded creators, as well as the Oppia development team, to ensure their creations are well-realized.
```

Comment:
```suggestion
              The <a [href]="TEACH_FORM_URL"><em>Teach with Oppia</em></a> program invites applications from volunteers who can demonstrate a passion and aptitude for helping students learn basic mathematics. Participants will receive personalized feedback on the lessons they create, as well as assistance with playtesting and promoting their lessons. They'll also be able to connect with other like-minded creators, as well as the Oppia development team, to ensure their creations are well-realized.
```
Can you see if this is working?

Reply:
Does work!
![Screenshot from 2020-07-16 12-17-02](https://user-images.githubusercontent.com/30462390/87636414-5aa77a80-c75e-11ea-8a06-c509d0392ad0.png)
![Screenshot from 2020-07-16 12-16-53](https://user-images.githubusercontent.com/30462390/87636420-5c713e00-c75e-11ea-95a3-5ff183af079e.png)

Reply:
Pushing!